plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -174,11 +174,7 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
             )
           )
         }
-        locationPuckManager?.let {
-          if (!it.isLayerInitialised()) {
-            it.initialize(style)
-          }
-        }
+        locationPuckManager?.initialize(style)
```

Comment:
could you cover the change with some unit tests?

Reply:
Yes, coming!

Reply:
Added test, ptal